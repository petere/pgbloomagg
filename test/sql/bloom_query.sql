SELECT bloom_query('one',   b'0110010000100110');
SELECT bloom_query('two',   b'0110010000100110');
SELECT bloom_query('three', b'0110010000100110');
SELECT bloom_query('four',  b'0110010000100110');
SELECT bloom_query('five',  b'0110010000100110');
SELECT bloom_query('six',   b'0110010000100110');
SELECT bloom_query('seven', b'0110010000100110');
SELECT bloom_query('eight', b'0110010000100110');
SELECT bloom_query('nine',  b'0110010000100110');
SELECT bloom_query('ten',   b'0110010000100110');

SELECT bloom_query('foo', b'');

SELECT 'one' << b'0110010000100110';
