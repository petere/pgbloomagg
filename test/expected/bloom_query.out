SELECT bloom_query('one',   b'0110010000100110');
 bloom_query 
-------------
 t
(1 row)

SELECT bloom_query('two',   b'0110010000100110');
 bloom_query 
-------------
 t
(1 row)

SELECT bloom_query('three', b'0110010000100110');
 bloom_query 
-------------
 t
(1 row)

SELECT bloom_query('four',  b'0110010000100110');
 bloom_query 
-------------
 t
(1 row)

SELECT bloom_query('five',  b'0110010000100110');
 bloom_query 
-------------
 t
(1 row)

SELECT bloom_query('six',   b'0110010000100110');
 bloom_query 
-------------
 t
(1 row)

SELECT bloom_query('seven', b'0110010000100110');
 bloom_query 
-------------
 t
(1 row)

SELECT bloom_query('eight', b'0110010000100110');
 bloom_query 
-------------
 f
(1 row)

SELECT bloom_query('nine',  b'0110010000100110');
 bloom_query 
-------------
 t
(1 row)

SELECT bloom_query('ten',   b'0110010000100110');
 bloom_query 
-------------
 f
(1 row)

SELECT bloom_query('foo', b'');
ERROR:  division by zero
CONTEXT:  SQL function "bloom_query" statement 1
SELECT 'one' << b'0110010000100110';
 ?column? 
----------
 t
(1 row)

